astropy/utils/tests/test_data.py

```diff
@@ -464,11 +476,3 @@ def test_path_objects_get_readable_fileobj():
     with get_readable_fileobj(fpath) as f:
         assert f.read().rstrip() == ('This file is used in the test_local_data_* '
                                      'testing functions\nCONTENT')
-
-
-@pytest.mark.remote_data('astropy')
-def test_get_cached_urls():
```

Comment:
Note: This is now indirectly tested in `test_download_parallel` above. No need to have duplicate tests that do the same thing.

Reply:
Maybe just note in a comment in `test_download_parallel` that it does this?  Just want to make sure someone in the future who might decide to edit `test_download_parallel` knows that it should also be checking this